specutils/manipulation/extract_spectral_region.py

```diff
@@ -149,7 +149,11 @@ def extract_region(spectrum, region):
             if left_index > right_index:
                 left_index, right_index = right_index, left_index
 
-            extracted_spectrum.append(spectrum[left_index:right_index])
+            if len(spectrum.flux.shape)==1:
+                extracted_spectrum.append(spectrum[left_index:right_index])
+            else:
+                extracted_spectrum.append(spectrum[..., left_index:right_index])
+
```

Comment:
```suggestion
            extracted_spectrum.append(spectrum[..., left_index:right_index])
```
See suggestion for `Spectrum1D.__getitem__`, which would obviate the need for the test.
Note: double blank lines only between function definitions (E303)

Reply:
Good call, much cleaner